.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,29 @@
+name: e2e
+on:
+  pull_request:
+    paths-ignore:
+    - '**/*.md'
+  schedule:
+  - cron: '0 0 * * *' # daily to pick up releases
+  workflow_dispatch:
+jobs:
+  kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.17'
+    - name: Install kind
+      run: |
+        curl -sLo kind "$(curl -sL https://api.github.com/repos/kubernetes-sigs/kind/releases/latest | jq -r '[.assets[] | select(.name == "kind-linux-amd64")] | first | .browser_download_url')"
+        chmod +x kind
+        sudo mv kind /bin/
+    - name: Create kind cluster
+      run: |
+        kind create cluster
+        kind export kubeconfig
+    - name: Start Combo
+      run: make run
+    - name: Run e2e suite
+      run: make test-e2e
```

Comment:
My personal preference is to expose knobs for e2e suites to handle the installation of the stack (e.g. e2e-local vs. e2e targets for OLM's Makefile), or to simply just run the e2e suite using an existing cluster, but this is fine for now.

Reply:
I had a pre-req for this originally on the `run` target but got rid of it for fear of bloat. Do you think I should go ahead and add that back? Then I would probably also add another target I got rid of called `test-e2e-local` for running locally unless that is too much. Thoughts?

Reply:
Okay, I'm pushing up some changes that should address this. Let me know what you think.